torch/_inductor/kernel/flex_attention.py

```diff
@@ -793,9 +800,6 @@ def flex_attention_backward_grid(
     sparse_idx_z = off_z % SM_Z
     sparse_idx_h = off_h % SM_H
 
-    SPARSE_Q_BLOCK_CNT = Q_LEN // SPARSE_Q_BLOCK_SIZE
-    SPARSE_KV_BLOCK_CNT = KV_LEN // SPARSE_KV_BLOCK_SIZE
-
     sparse_hz_offset = sparse_idx_z * SM_H + sparse_idx_h
```

Comment:
should this use stride as well?

Reply:
mmm... I don't want to use stride here since anyways we're assuming all of the blocksparse data structures are contiguous (and have the same size ... actually).